### DISCUSSION TITLE
No definition for read and write in API docs

### DISCUSSION THREAD

**Discussion by beaka** [pschatzmann/arduino-audiokit#108] [Q&A] [answered]
Hi, In the code there is functions to read and write eg:
kit.read(buffer, BUFFER_SIZE);
kit.write(bufferout, len);
but I cannot find a reference to them in the docs.
What am I missing to not find them?

**Comment by pschatzmann** [comment, marked as answer]
The documentation shows only the methods which are available in all platforms: This was part of the original functionality which has been provided for the ESP32 only. 

However __this library__ should be platform independent, so I decided to remove it in the long run.

This decision makes also sense since the I2S API will change in the latest IDF versions and I want to have to maintain this only in one single project (=AudioTools)
